lib/busser/runner_plugin/serverspec.rb

```diff
@@ -56,9 +56,8 @@ def run_bundle_install
   def install_serverspec
     Gem::Specification.reset
     if Array(Gem::Specification.find_all_by_name('serverspec')).size == 0
-      if Gem::Version.new(RUBY_VERSION.dup) < Gem::Version.new('2.0')
-        banner('Installing net-ssh < 2.10')
-        install_gem('net-ssh', '< 2.10')
+      if Gem::Version.new(RUBY_VERSION.dup) < Gem::Version.new('2.3')
```

Comment:
this can no longer happen since the gem will not be installed on ruby <2.3 ... so might as well delete that